include/dt-bindings/dma/stm32_dma.h

```diff
@@ -1,16 +1,44 @@
 /*
  * Copyright (c) 2019 Song Qiang <songqiang1304521@gmail.com>
+ * Copyright (c) 2020 STMicroelectronics
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_DMA_STM32_DMA_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_DMA_STM32_DMA_H_
+
 /* macros for channel-config */
+#if defined(CONFIG_DMA_STM32_V1)
```

Comment:
Do we need to differentiate V1 vs V2?
There's no difference made in "st,stm32-dma.yaml " binding

Reply:
the only diff is about the fifo threshold for V1 (V2 has not)